### PR TITLE
feat: long press custom preview component

### PR DESCRIPTION
**Pull request by brainbicycle**
This PR resolves [PHIRE-115] 

### Description

Design feedback on long press:
- Use same full width card everywhere
- Hide heart icon
- Show gallery name

design-feedback-preview

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

- Long press menu preview design feedback - Brian, George

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[PHIRE-115]: https://artsyproduct.atlassian.net/browse/PHIRE-115?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 6a7c594685d2be1af4578d7ecfdb45fc8be9f535
</p>

File: package.json [hunk 1]
```diff
@@ -172,7 +172,7 @@
     "react-native-appboy-sdk": "1.35.1",
     "react-native-bootsplash": "3.2.0",
     "react-native-config": "https://github.com/artsy/react-native-config.git#v1.4.12-artsy",
-    "react-native-context-menu-view": "git+https://github.com/artsy/react-native-context-menu-view.git#v1.9.4-artsy",
+    "react-native-context-menu-view": "git+https://github.com/artsy/react-native-context-menu-view.git#v1.10.10-artsy",
```
Comment (gkartalis) [inline package.json:175]: 👏

File: src/app/Components/ArtworkGrids/ArtworkGridItem.tsx [hunk 2]
```diff
@@ -438,7 +438,10 @@ export const saleMessageOrBidInfo = ({
 export default createFragmentContainer(Artwork, {
   artwork: graphql`
     fragment ArtworkGridItem_artwork on Artwork
-    @argumentDefinitions(includeAllImages: { type: "Boolean", defaultValue: false }) {
+    @argumentDefinitions(
+      includeAllImages: { type: "Boolean", defaultValue: false }
+      width: { type: "Int" }
+    ) {
```
Comment (gkartalis) [inline src/app/Components/ArtworkGrids/ArtworkGridItem.tsx:444]: 👀 do we actually need this?

**Review by gkartalis — APPROVED** [review]

File: src/app/Components/ArtworkGrids/ArtworkGridItem.tsx (hunk 2 above)
Comment (brainbicycle) [reply src/app/Components/ArtworkGrids/ArtworkGridItem.tsx:444]: yeah we need it so the artwork passed down to the preview component has all the resized image types attached 👀

**Pull request merged**